2.0/s2i/bin/run

```diff
@@ -10,6 +10,16 @@ set -e
 export RACK_ENV=${RACK_ENV:-"production"}
 
 if is_puma_installed; then
+  export_str=$(cgroup-limits.py)
+  [[ -n "$export_str" ]] && export $export_str
+
+  if [[ -z "${MEMORY_LIMIT_IN_BYTES:-}" ]]; then
+    echo "Warning: Can't detect memory limit"
+  fi
+  if [[ -z "${NUMBER_OF_CORES:-}" ]]; then
+    echo "Warning: Can't detect number of cores"
```

Comment:
@mnagy if the script error out, we should display the traceback and exit. if the script return nothing (that will never happen, because we always print the max memory var), we just export it. if the vars are empty (never happen because the files are always there with some default value) it means the script failed and we will never get here :-)